### DISCUSSION TITLE
Native LLM support

### DISCUSSION THREAD

**Discussion by snwfdhmp** [snwfdhmp/llm#2] [General]
# Native LLM support

## Goal

Define what LLMs should be included in the native repo.

> Important: We're not talking about pre-downloading models, just talking about "interface implementation" with installation/usage/parameters implementation.

## Format

```
- [modelName](url): Why it should be added
```

## Vote

Vote for each model by adding 👍 👎 on the message. 👎  requires adding a comment to explain why.

## Models included yet

- [gpt-3.5-turbo](#): Top famous models
- [text-davinci-003](#): Top famous models
- [GPT-4 web](#): Top famous models

**Comment by snwfdhmp** [comment]
- [Bing Chat](https://www.bing.com/search?q=Bing+AI&showconv=1): Famous model. Allows to use GPT-4 for free.

**Comment by snwfdhmp** [comment]
- [Google Bard](https://bard.google.com/?hl=en): Top famous model by google.

**Comment by bionicles** [comment]
claude would be a good one to add here
https://console.anthropic.com/docs/api
